src/commands/serviceConnector/createServiceConnector.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ICreateLinkerContext, createLinker } from "@microsoft/vscode-azext-serviceconnector";
+import { AzureWizard, AzureWizardPromptStep, createSubscriptionContext, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { ContainerAppItem } from "../../tree/ContainerAppItem";
+import { createActivityContext } from "../../utils/activityUtils";
+import { localize } from "../../utils/localize";
+import { pickContainerApp } from "../../utils/pickContainerApp";
+import { ContainerPickStep } from "./ContainerPickStep";
+import { IServiceConnectorWithActivityContext } from "./IServiceConnectorContext";
+
+export async function createServiceConnector(context: ICreateLinkerContext, item?: ContainerAppItem): Promise<void> {
+    if (!item) {
+        item = await pickContainerApp(context);
+    }
+    const { subscription, containerApp } = item;
+
+    const createServiceTitle: string = localize('createServiceConnector', 'Create Service Connector');
+
+    const wizardContext: IServiceConnectorWithActivityContext = {
+        activityTitle: createServiceTitle,
+        ...context,
+        ...createSubscriptionContext(subscription),
+        containerApp: containerApp,
+        sourceResourceUri: containerApp.id,
+        ...(await createActivityContext())
+    }
+
+    const promptSteps: AzureWizardPromptStep<IServiceConnectorWithActivityContext>[] = [
+        new ContainerPickStep(),
+    ];
+
+    const wizard: AzureWizard<IServiceConnectorWithActivityContext> = new AzureWizard(wizardContext, {
+        title: localize('createServiceConnector', 'Create Service Connector'),
+        promptSteps,
+    });
+
+    await wizard.prompt();
+    await createLinker(wizardContext, nonNullValue(item));
```

Comment:
I see `createLinker` builds its own wizard.  Is the intent that whenever we need to create a link that we must first add a step to define the scope?  

If so, I'm wondering if it makes more sense to use a different pattern here.  An example solution would be to build these functions such that you can pass them the precursor step(s), and they incorporate it into the wizard for you.  Depending on the answer above, there are probably some other patterns that could work as well... mostly just positing for discussion purposes since I don't have a strong opinion on the solution yet.

(Main reason I brought this was up... something felt weird to me about having two wizards going like this... especially since the first one builds for the activity log, but `ICreateServiceContext` doesn't type/build for an activity context.  I was thinking it made more sense to try and design around building just one continuous wizard process if possible in these instances, whether that be through using a helper to build the wizard like suggested above, or through importing the steps into the wizard used in the client extension... )